superset-frontend/src/views/CRUD/data/dataset/DatasetPage/DatasetPage.test.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { render, screen } from 'spec/helpers/testing-library';
+import DatasetPage from 'src/views/CRUD/data/dataset/DatasetPage';
+
+describe('DatasetPage', () => {
```

Comment:
- DatasetPage is the whole "add dataset" page
- DatasetLayout structures all of the components on the page.
- DatasetPanel is the center component currently holding the "Select Dataset Source" blank state

I think if we were to rename them I'd suggest something like:
- DatasetPage -> AddDatasetPage
  - Since there will also be an EditDatasetPage
- DatasetLayout -> Either DatasetPageLayout or keep it as is, I think it describes it properly
- DatasetPanel -> Could probably leave this name as is as well, but I'm open to suggestions 😁

Reply:
Ok, I think that makes sense. Maybe we should make DatasetPage into just AddDataset. I also think that we should maybe put DatasetLayout into its own folder, with an index.tsx and a testing file.

Reply:
Good call, changed in [`this commit`](https://github.com/apache/superset/pull/21058/commits/68867b2dc071c54e33408ed95faf8f17ec021225).